nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/marshallers/DocumentMarshaller.java

```diff
@@ -91,10 +92,12 @@ protected static Document readDocument(JsonParser jp) throws IOException {
                 readProperties(jp, properties);
             } else if ("changeToken".equals(key)) {
                 changeToken = jp.getText();
+            } else if ("isProxy".equals(key)) {
+                isProxy = jp.getText();
```

Comment:
Why not getting the `isProxy` field directly as a boolean? We know it must be present, and that's a boolean. (yes it's not consistent with the other boolean fields but we can start here :)